src/FOMObot/Helpers/Bot.hs

```diff
@@ -56,3 +63,21 @@ alertFOMOChannel channelID = do
     Slack.sendMessage fomoChannel message
   where
     message = "Check out <#" <> (channelID ^. Slack.getId) <> ">"
+
+processCommand :: Slack.Event -> Bot ()
```

Comment:
As Josh pointed out, I think `Bot.hs` is becoming a god file. There's a lot in this file.

What about extracting the new code to `FOMObot.Helpers.Preferences`?